docs/playbooks/wind-river-cloud-platform-deployment-manager.yaml

```diff
@@ -744,6 +744,7 @@
               timeout: "{{ boot_wait_time }}"
             register: waiting_after_reboot
             ignore_unreachable: true
+            ignore_errors: true
```

Comment:
Please add comment here to explain why ignore_unreachable and ignore_errors are both expected

Reply:
Different scenarios were seen and observed that the task could be marked as failed sometimes and could be marked as unreachable. It could be something specific of ansible that we might need to investigate further. But in this case, with both conditions, we cover the  possible scenarios.

Reply:
Can you add two comments regarding the cases "ignore_unreachable" and "ignore_errors" respectively for future reference and understanding?

Reply:
It can ignore a task failure due to the host instance being ’UNREACHABLE’ with the ignore_unreachable keyword. Ansible ignores the task errors but continues to execute future tasks against the unreachable host.
It can ignore errors when the task is failed, except those syntax errors, undefined variable errors, connection failures, execution issues.